lib/rules/new-module-imports.js

```diff
@@ -33,6 +33,24 @@ module.exports = {
   },
 
   create(context) {
+    function reportNestedProperties(properties, parent) {
+      properties.forEach(item => {
+        const match = GLOBALS[`Ember.${parent}.${item.key.name}`];
```

Comment:
is hard coding `Ember.` here correct when this is also used for data rules?

Reply:
I've extracted some `new-module-imports` rule internals to `lib/utils/new-module.js`. This is why some lines change here. I did that to use the `buildMessage` function in the new rule `new-ember-data-packages` (btw, should we rename the rule as well or is it fine?).

So there are now two rules:
- `new-module-imports`: it applies "New Module Imports" from Ember RFC #176
- `new-ember-data-packages`: it applies "Ember Data Packages" from Ember RFC #395

Harcoding `Ember` is fine here. :)

The